kinetic/distribution.yaml

```diff
@@ -9010,6 +9010,14 @@ repositories:
         release: release/kinetic/{package}/{version}
       url: https://github.com/Terabee/teraranger-release.git
       version: 1.0.0-1
+  teraranger-array:
```

Comment:
I would suggest to use an underscore here to to keep it consistent. That would also make the package lost obsolete since the name of the only package matches the repository name.